templates/web/base/hercules/bin_days.html

```diff
@@ -0,0 +1,76 @@
+[% SET bodyclass = 'hercules' %]
+[% USE date(format = '%A, %-d~~~ %B') %]
+
+[% INCLUDE header.html %]
+
+<h1 class="govuk-heading-xl">Your bin days</h1>
+
+<dl class="hercules__address">
+  <dt class="hercules__address__title">Address</dt>
+  <dd class="hercules__address__property">[% property.Description | title %]</dd>
+</dl>
+<div class="hercules__collections">
+  <h2 class="govuk-heading-l">Your collections</h2>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">  
+
+[% FOR unit IN service_data %]
+
+<a href="#" class="hercules-service-name-link"><h3 class="govuk-heading-m">[% unit.service_name %]</h3></a>
```

Comment:
I am not sure that clicking on this title to expand out the list of options is at all obvious.

Reply:
Have just pushed it. FYI this currently doesn't work without JS